src/api/v1/community_businesses/volunteers/__tests__/projects.test.integration.ts

```diff
@@ -31,7 +31,7 @@ describe('API /community-businesses/me/volunteers/projects', () => {
         method: 'GET',
         url: '/v1/community-businesses/me/volunteers/projects',
         credentials: {
-          scope: ['organisation_details-parent:read'],
+          scope: ['organisations_volunteer-parent:read'],
```

Comment:
`organisations_volunteer` -> `organisations_volunteers`

all other permissions are plural.

Can you also add organisations_volunteers to `docs/permissions.md`